Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallContentCommand.cs

```diff
@@ -185,7 +186,7 @@ private IUnarchiver CreateUnrachiver(string destinationDir, MapHashExtractedFile
             }
         }
 
-        private void InstallFile(SourceFile sourceFile, CancellationToken cancellationToken)
+        private void InstallFile(SourceFile sourceFile, CancellationToken cancellationToken, bool lastEntry)
```

Comment:
Please rename `lastEntry` to `isLastEntry`.

Reply:
Done